src/drivers/intel/cavs/ipc.c

```diff
@@ -172,9 +172,6 @@ int ipc_platform_compact_read_msg(ipc_cmd_hdr *hdr, int words)
 
 enum task_state ipc_platform_do_cmd(void *data)
 {
-#if !CONFIG_SUECREEK
```

Comment:
Is this patch in another PR ?

Reply:
yes, I closed #4469

Reply:
@lgirdwood what do you think about this change - does it seem safe to you? @ranj063 expressed some concerns in https://github.com/thesofproject/sof/pull/4469#discussion_r667016011